kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -384,4 +365,32 @@ public Subscription create(Subscription subscription) {
     }
 
 
+    @Override
+    public final void createSubscriptionPacks() {
+        if (subscriptionPackDataService.byName("childPack") == null) {
+            createSubscriptionPack("childPack", SubscriptionPackType.CHILD, CHILD_PACK_WEEKS, 1);
+        }
+        if (subscriptionPackDataService.byName("pregnancyPack") == null) {
+            createSubscriptionPack("pregnancyPack", SubscriptionPackType.PREGNANCY, PREGNANCY_PACK_WEEKS, 2);
+        }
+    }
+
+
+    private void createSubscriptionPack(String name, SubscriptionPackType type, int weeks,
```

Comment:
Right -- these methods are no longer called in the ctor (that's what I re-removed in this commit). But they still exist here because they're called by both API and Kilkari test code. Happy to move them if you have a better idea of where to put them... (they show up as green in this diff just because I moved them from the top to the bottom of the same file).